x/storage/client/cli/tx.go

```diff
@@ -823,11 +827,14 @@ func CmdDiscontinueBucket() *cobra.Command {
 
 func CmdMirrorObject() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "mirror-object [object-id]",
+		Use:   "mirror-object [object-id] [bucket-name] [object-name]",
```

Comment:
So is this, for the optional param, we should not make it as a must.

Reply:
This one also uses flags now.